### ISSUE TITLE
[feature request] 对于element-plus1.2.0之后的版本，el-data-picker组件将在vite打包后消失

### ISSUE THREAD

**Issue by SmallSample** [element-plus/element-plus#4266]
### Existing Component
Yes

### Component Name
el-date-picker

### Description
1.我用vue-cli打包与vite打包作对比，发现vite打包后会出现vue警告：[Vue warn]: Component is missing template or render function
2.element-plus1.2.0以前的版本用vite打包后，el-date-picker组件正常显示

**Comment by odex21** [comment]
That's a workaround
```ts
const config = {
    build: {
      rollupOptions: {
        output: {
          manualChunks: {
            elementPlus: ['element-plus'],
          },
        },
      },
    },
}
```
This seems to be related to three shaking.

**Comment by SmallSample** [comment]
https://github.com/SmallSample/demo @msidolphin

**Comment by msidolphin** [comment]
![image](https://user-images.githubusercontent.com/26672484/140697196-0aeb5675-64e9-45a2-aca3-f97751fe0a2e.png)

It seems to be importing the incorrect files, you can temporarily change them to `.mjs`

**Comment by MrFu1998** [comment]
直接拉取  [element-plus-vite-starter](https://github.com/element-plus/element-plus-vite-starter) 仓库代码，通过 `yarn` 安装后直接 `build`， 然后 `el-date-picker` 确实消失了

**Comment by SmallSample** [comment]
How to operate it? @msidolphin

**Comment by msidolphin** [comment]
> How to operate it? @msidolphin
```diff
- import script$3 from '../../time-picker/src/common/picker.vue_vue&type=script&lang.mjs';
+ import script$3 from '../../time-picker/src/common/picker.mjs';
```

**Comment by SmallSample** [comment]
Thank you. It has been solved @msidolphin

**Comment by SmallSample** [comment]
Normal display, but there will still be Vue warnings @msidolphin 
![image](https://user-images.githubusercontent.com/34450888/140844692-9af10bcd-8195-40fe-8d14-554006c0939b.png)

**Comment by czq297297** [comment]
+1

**Comment by msidolphin** [comment]
If you don't care about packing volume, you can turn `treeshake` off or import the commonjs module

**Comment by sxzz** [comment]
Fixed in #4310. We will release it ASAP.
